src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.js

```diff
@@ -155,6 +155,23 @@ export function transformFormToCCRequest(state) {
         providerZipCode1: provider.address.postalCode,
       },
     ];
+  } else if (
```

Comment:
Looking over Tony’s previous bug fix, he eliminated `hasResidentialAddress` as a search criteria altogether and replaced it with `featureCCIteration`. I think he forgot to carry that over to this transformer file.

Replacing Line 119 with `const featureCCIteration = selectFeatureCCIterations(state)` will introduce the CCIteration toggle flag.

Replace Line 124 from `hasResidentialAddress` to `featureCCIteration` will set the right logic and no need to add this if statement block

FYI - Line 139 is a boolean that's true only under the CCIteration toggle is off and user has manually entered provider information form.

Reply:
@jenniemc Thank you for the detailed suggestions on this fix. It really helped to understand the feature and where it should be evaluated.